ndbench-core/src/main/java/com/netflix/ndbench/core/defaultimpl/NdBenchGuiceModule.java

```diff
@@ -39,8 +40,14 @@
     protected void configure()
     {
         bind(NdBenchMonitor.class).to(FakeMonitor.class);
-        //bind(IClusterDiscovery.class).to(AWSLocalClusterDiscovery.class);
-        bind(IClusterDiscovery.class).to(LocalClusterDiscovery.class);
+        String discoveryEnv = System.getenv(NdBenchConstants.DISCOVERY_ENV);
```

Comment:
Can you update the wiki on how to use this new feature, if you could put few directions on configuring IClusterDiscovery it would be helpful.

Wiki - https://github.com/Netflix/ndbench/wiki/Configuration#default-implementation